src/pybind/mgr/dashboard_v2/tests/helper.py

```diff
@@ -32,8 +33,15 @@ def _delete(self, url, data=None):
     def _put(self, url, data=None):
         self._request(url, 'PUT', data)
 
-    def assertJsonBody(self, data):
-        self.assertBody(json.dumps(data))
+    def assertJsonBody(self, data, msg=None):
```

Comment:
why do you need the `msg` parameter, it's not being used anywhere AFAICT

Reply:
Copy & paste of the existing code. I didn't want to mess with the logic in detail. Instead, leave it as it is.

Reply:
C&P from where?

Reply:
`cheroot.test.webtest.WebCase#assertBody`